app/server/api-generator/src/test/java/io/syndesis/server/api/generator/openapi/BaseOpenApiGeneratorExampleTest.java

```diff
@@ -61,6 +61,7 @@
     protected final String specification;
 
     public BaseOpenApiGeneratorExampleTest(final String connectorQualifier, final String name, final String version) throws IOException {
+        System.out.println(name);
```

Comment:
```suggestion
```

Reply:
Oh did that for debugging reasons. Thanks!